models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -89,6 +89,7 @@ FROM
     ("crv-curve-dao-token", "ethereum", "CRV", "0xd533a949740bb3306d119cc777fa900ba034cd52", 18),
     ("ctxc-cortex", "ethereum", "CTXC", "0xea11755ae41d889ceec39a63e6ff75a02bc1c00d", 18),
     ("cusdt-compound-usdt", "ethereum", "cUSDT", "0xf650c3d88d12db855b8bf7d11be6c55a4e07dcc9", 8),
+    ("cusdc-compound-usd-coin", "ethereum", "cUSDC", "0x39aa39c021dfbae8fac545936693ac917d5e7563", 8),
```

Comment:
Hi @fahimahmedx , the token-id is legit, however, it seems this api/token is unactive:
https://api.coinpaprika.com/v1/coins/cusdc-compound-usd-coin
Which means the prices.usd will not work for this token even if this PR get merged.

Reply:
Historical data is needed and it updates with previous days data which is enough for me to work with for the dashboard.

Reply:
In this case do we need to merge an inactive token to `prices.usd` for its historical data? @dot2dotseurat

Reply:
thanks for following up @Hosuke -- let's confirm with @aalan3 on the question here. if we merge in this token with an inactive id, will it load historical prices? then just ignore moving forward? @fahimahmedx would like to see historical data, if possible

Reply:
trying @aalan3 one more time for the question 🙏

Reply:
Sorry for the late reply, I think this will be fine. Let's try it.